src/commands/notify.yml

```diff
@@ -67,6 +67,16 @@ parameters:
     type: boolean
     default: true
 
+  include_dashboard:
```

Comment:
No need for this since you can use dashboard_link for the conditional

Reply:
updated